data/migrations/20190719084726_projects.js

```diff
@@ -0,0 +1,43 @@
+
+    exports.up = function(knex) {
+        return knex.schema
+            .createTable("projects", tbl => {
+                tbl.increments()
+                tbl
+                .string("name")
+                .unique()
+                .notNullable()
+                tbl
+                .string("description")
+                .notNullable()
+                tbl
```

Comment:
Instead of beginning each field with `null` you can chain `.defaultTo(false)` at the end to default the boolean value to false.